cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -5,15 +5,16 @@
 from cfme import test_requirements
 from cfme.utils.appliance.implementations.ui import navigate_to
 from cfme.utils.update import update
-from cfme.utils.version import current_version
 from cfme.utils.wait import wait_for
 
 pytestmark = [
     pytest.mark.long_running,
     pytest.mark.meta(server_roles=["+embedded_ansible"]),
-    pytest.mark.uncollectif(lambda: current_version() < "5.8"),
+    pytest.mark.uncollectif(lambda appliance: appliance.version < "5.8"),
```

Comment:
Required: Please add a reason info for this uncollectif, also update the same places below